src/libsyntax/print/pprust.rs

```diff
@@ -1909,6 +1911,10 @@ impl<'a> State<'a> {
                 try!(self.pclose());
             }
             ast::ExprMac(ref m) => try!(self.print_mac(m, token::Paren)),
+            ast::ExprCompletion(ref e) => {
+                try!(self.print_expr(&**e));
+                try!(word(&mut self.s, ".\u{B6}"));
```

Comment:
Indeed, pretty much a perfect example of a bikeshed. Anyhow, font support is a good reason to choose pilcrow over caret. Carry on!